### ISSUE TITLE
As an Admin I want to delete user account

### ISSUE THREAD

**Issue by BorderITS** [borderitsolutions/amadaa#83]

**Comment by lcabrini** [comment]
Yes fine, but if we delete the account from the database, any linked information would also either go or become orphaned. We probably would not actually delete the user, just apply a "deleted" flag.

**Comment by BorderITS** [comment]
okay
